uber/templates/regform.html

```diff
@@ -82,7 +82,7 @@
     </div>
 {% endif %}
 
```

Comment:
This fix is to just always have the ``amount_extra`` field, which is fine since we hide it anyway and don't show the badge row on dealer applications.  I've tested this on several of the forms (main prereg form, dealer prereg form, confirm form, unassigned group badge form) and it works as expected.